ci/ciRunsDockerContainer/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-  build-essential git python3-pip python3.10-venv openssl make curl openssh-client gettext-base \
+  gettext-base unzip wget apt-utils build-essential git python3-pip python3.10-venv openssl make curl openssh-client gettext-base gnupg software-properties-common \
```

Comment:
I would recommend separating each entry with ```\``` and sorting them alphabetically for easing comparison later on.

Reply:
will do thxx